### PR TITLE
Enable usage of non-fractional currencies

### PR DESCRIPTION
**Pull request by najafimh**
#108

### CODE REVIEW

File: subscriptions/conf.py [hunk 1]
```diff
@@ -24,9 +24,10 @@ class Currency():
             n_sep_by_space (bool): Whether the currency symbol is
                 separated from negative values by a space.
             mon_decimal_point (str): The character used for decimal
-                points.
+                points. Use 0 if the currency doesn't need fraction.
```
Comment (studybuffalo) [inline subscriptions/conf.py]: Minor text fix: can we change this to "Use 0 if this is a non-decimal currency."

File: subscriptions/conf.py [hunk 2]
```diff
@@ -24,9 +24,10 @@ class Currency():
...
             mon_thousands_sep (str): The character used for separating
-                groups of numbers.
+                groups of numbers. Use 0 if the currency doesn't need
```
Comment (studybuffalo) [inline subscriptions/conf.py]: Sorry, I am not sure I am understanding this change. This argument refers to which separator is used for visually separating groups of numbers and not actually used in any fractional/decimal sense.

For example, we use `,` in USD to represent `$1,000,000.00`

I think we can just remove these changes?

File: subscriptions/conf.py (hunk 1 above)
Comment (najafimh) [reply subscriptions/conf.py]: Yes it would be better.

File: subscriptions/conf.py (hunk 2 above)
Comment (najafimh) [reply subscriptions/conf.py]: I thought that was for international format
Yeah that's not needed actually

**Comment by najafimh** [comment]
I applied all the fixes needed. It should be ready

**Comment by studybuffalo** [comment]
Hey @najafimh - thanks for all the work on this. It is looking like things are functioning as intended. I will merge this into master and do a minor version update as soon as I can.

**Comment by najafimh** [comment]
You’re welcome :) I’m developing my own website based on your library, so I’m probably coming back for next changes. Thanks for your awesome library

**Pull request merged**